SilKit/include/silkit/capi/Lin.h

```diff
@@ -161,7 +161,7 @@ typedef uint8_t SilKit_LinFrameStatus;
 typedef uint8_t SilKit_LinDataLength;
 
 //! \brief If configured for reception with this value, the data length validation of incoming frames is skipped.
-const SilKit_LinDataLength SilKit_LinDataLengthUnknown = 255u;
```

Comment:
oh boy. what an oversight....
is this symbol by default hidden? it's not visible/dllexported, right?
so recompiling with this fix should be fine for all users.